### PR TITLE
add : review system

### PR DESCRIPTION
**Pull request by OmkarK45**
# Reviewer Dashboard

### Changes
No changes in User Side
- Adds routes for a reviewer, 
- Adds routes for admin 

### Screenshots or Video GIFs
image

image

image

image

Features:
- A reviewer can: review a portfolio, request one for review and mark it as either ready or NOT ready.
- Menu Options for navigation bar for reviewers. 
- A reviewer can see their previously reviewer portfolios.

### CODE REVIEW

File: client/src/pages/auth/admin/login.tsx [hunk 1]
```diff
@@ -173,20 +232,19 @@ export default function Login() {
...
-                          checkboxToggle.guideCheck === true &&
-                          checkboxToggle.responsibitiesCheck === true
-                            ? false
-                            : true
-                        }
-                      >
-                        Login
-                      </Button>
+                      <Field>
+                        {({ form }: { form: any }) => (
```
Comment (SAYUK09) [inline client/src/pages/auth/admin/login.tsx:236]: Can we have an appropriate type here.

File: client/src/pages/dashboard/index.tsx [hunk 2]
```diff
@@ -10,14 +10,18 @@ import {
...
+import { useAuth } from '../../context/AuthContext'
 
 function Dashboard() {
   const [currentStatus, setCurrentStatus] = useState('portfolio_not_submitted')
   const [submissionNo, setSubmissionNo] = useState(null)
   const toast = useToast()
+  const router = useRouter()
+  const { setAuthState } = useAuth()
   useEffect(() => {
     async function fetch() {
```
Comment (SAYUK09) [inline client/src/pages/dashboard/index.tsx:26]: Why aren't we using an iife here?

**Review by SAYUK09 — CHANGES_REQUESTED** [review]
Everything looks great. Have some doubts, commented them

File: client/src/pages/auth/admin/login.tsx (hunk 1 above)
Comment (OmkarK45) [reply client/src/pages/auth/admin/login.tsx:236]: Fair point. However Formik still has not added types for render props 
https://github.com/formium/formik/issues/2086#issuecomment-835856021

File: client/src/pages/dashboard/index.tsx (hunk 2 above)
Comment (OmkarK45) [reply client/src/pages/dashboard/index.tsx:26]: TBH it doesn't make any difference

Comment (SAYUK09) [reply client/src/pages/dashboard/index.tsx:26]: Oh! Thanks. didn't know about this.

File: client/src/pages/auth/admin/login.tsx (hunk 1 above)
Comment (SAYUK09) [reply client/src/pages/auth/admin/login.tsx:236]: Oh. I see.

**Comment by MehulKChaudhari** [comment]
Hey Omkar, Tested locally Works as it should be. 
Awesome work bro 🔥

**Review by MehulKChaudhari — APPROVED** [review]
LGTM 🔥

**Comment by sohamsshah** [comment]
Hi Omkar, the codebase looks great, clean and well implemented.
Have raised few issues #174 , #175 , #176 , #177 and #178 for the same. Can resolve them based on priority-basis.

**Pull request merged**